### ISSUE TITLE
ci test on master failed

### ISSUE THREAD

**Issue by clemens-tolboom** [molgenis/molgenis-service-armadillo#636]
The CI step on `master` fails on `Start docker-compose and wait for `release-test.R` to finish`

The PR of #563 CI tests were successful. So why is this?

Happened on newly effort for PR #649

```log
── Resource upload ──

ℹ Creating project [udlkiq6cfg]
Created project 'udlkiq6cfg' without users
ℹ Uploading resource file to http://armadillo:8080/ into project [udlkiq6cfg]
⠏   user  system elapsed 
  0.671   0.239 101.238 
ℹ Creating resource

── Creating linked view on table ──

Created project 'ueu82kylfd' without users
✖ Unable to create linked object ueu82kylfd/core-variables/nonrep from source: lifecycle/core/nonrep, status code: 405
Error: Unable to create linked object ueu82kylfd/core-variables/nonrep from source: lifecycle/core/nonrep, status code: 405
Execution halted

Exited with code exit status 1
```

## SSH into

- is file permission doing something?
  - we run tests in docker-compose where we have workaround due

```yml
            # 1 CircleCI docker in docker workaround
            docker cp ./armadillo/config $ARMADILLO:/config
            docker cp ./armadillo/data   $ARMADILLO:/data
            docker cp ./armadillo/logs   $ARMADILLO:/logs
            # .1 CircleCI docker in docker workaround
```

and

```yml
            # 2 CircleCI docker in docker workaround
            # See what changed within Armadillo
            docker cp $ARMADILLO:/logs   ./armadillo/logs
            # .2 CircleCI docker in docker workaround
```

This last steps seem not to work as `./armadillo/logs/` is still empty ... is this last workaround not working?

**Comment by clemens-tolboom** [comment]
We can try to run the CI test locally following [docker/ci/README.md](https://github.com/molgenis/molgenis-service-armadillo/blob/master/docker/ci/README.md)

This seems harder than expected. This needs some docs tweaks

The correct directory is `build/docker/ci` to follow the doc mentioned above.

**Comment by clemens-tolboom** [comment]
After following better docs from PR #638 I have successfully ran locally the breaking CI part.

```
── Creating linked view on table ──

Created project '6t8w3k9ec1' without users
✖ Unable to create linked object 6t8w3k9ec1/core-variables/nonrep from source: lifecycle/core/nonrep, status code: 405
Error: Unable to create linked object 6t8w3k9ec1/core-variables/nonrep from source: lifecycle/core/nonrep, status code: 405
Execution halted

Exited with code exit status 1
```

CircleCI is broke **on the** PR #638 instead off on `master` while PR #563 ran successfully in itself.

**Comment by clemens-tolboom** [comment]
We reverted the PR and new PR #649 still breaks.

**Comment by clemens-tolboom** [comment]
Fixed through #660
